client/snap_op.go

```diff
@@ -56,11 +56,19 @@ func (client *Client) Install(name string, options *SnapOptions) (changeID strin
 	return client.doSnapAction("install", name, options)
 }
 
+func (client *Client) InstallMany(names []string) (changeID string, err error) {
```

Comment:
How do we pass the channel in?  I believe the daemon can handle this, and we should ship the option from the command line there.